lib/app/App.vue

```diff
@@ -1,7 +1,9 @@
 <template>
   <div id="__nuxt">
     <% if (loading) { %><nuxt-loading ref="loading"></nuxt-loading><% } %>
-    <component v-if="layout" :is="nuxt.err ? 'nuxt' : layout"></component>
+    <% if (layoutTransition) { %><transition name="<%= layoutTransition.name %>" mode="<%= layoutTransition.mode %>"><% } %>
```

Comment:
I believe we should always add the `<transition>` if we want to be able to add layoutTransition into layouts.

Reply:
Do you mean we should always have `<transition>` even if no `layoutTransition` is set (so remove the conditional render from the template)?

Or do you mean the `<transition>` should be added by the user into layout template? This would present an issue with the way layouts work at the moment. The only resolution would be a programatic transition, similar to the one used by pages in `nuxt-child`.

Reply:
I mean always  transition and have default layout transition (like for pages, so the user can add `.layout-enter` to add an animation :))